pkg/kubelet/pleg/generic.go

```diff
@@ -251,10 +251,30 @@ func (g *GenericPLEG) Relist() {
 	for pid := range g.podRecords {
 		oldPod := g.podRecords.getOld(pid)
 		pod := g.podRecords.getCurrent(pid)
+
+		var cachePodStatus *kubecontainer.PodStatus
```

Comment:
We would also need to calculate in the evented pleg, so make sure we don't miss it there.

Reply:
that's a good point. I will cover it in new coming commit